client/cl_config.lua

```diff
@@ -50,7 +50,7 @@ Config.GangMenus = {
         vector3(0, 0, 0),
     },
     ['ballas'] = {
-        vector3(0,0, 0),
+        vector3(-974.15, 62.61, 51.78),
```

Comment:
What is this vector?

Reply:
Because im testing but i re-write for vector3(0,0,0)